### DISCUSSION TITLE
Please add Coqui xtts as well.

### DISCUSSION THREAD

**Discussion by nothingness6** [rsxdalv/TTS-WebUI#237] [Ideas]
This app is amazing and if you add Coqui xtts, it'll be much greater!
Thank you!

**Comment by rsxdalv** [comment]
Thank you for the suggestion.

Unfortunately, XTTS license has problems. Therefore it's not open source.
For more reference, I have written longer responses to those who suggested
this before.

**Comment by nothingness6** [comment]
> Thank you for the suggestion. Unfortunately, XTTS license has problems. Therefore it's not open source. For more reference, I have written longer responses to those who suggested this before.
> […](#)
> On Tue, Jan 2, 2024, 7:47 AM nothingness6 ***@***.***> wrote: This app is amazing and if you add Coqui xtts, it'll be much greater! Thank you! — Reply to this email directly, view it on GitHub <#237>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABTRXIY6EYTK3SI6IY3L6FTYMONQBAVCNFSM6AAAAABBJTDR3KVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGAZDANJWGY> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Oh, no! I didn't realize that. Thx for the quick reply, btw.
